logging/ContextLogPropertiesWrapper.ts

```diff
@@ -0,0 +1,18 @@
+import {LogPropertiesWrapper, PolarisLogProperties} from "@enigmatis/polaris-logs";
+import {GraphQLLogProperties} from "./GraphQLLogProperties";
+
+export class ContextLogPropertiesWrapper implements LogPropertiesWrapper{
```

Comment:
determine if LogPropertiesWrapper is necessary if not remove it
if its needed i dont think that ``shouldLogRequestAndResponse`` should be here

Reply:
deleted for now